src/core/toga/widgets/textinput.py

```diff
@@ -178,6 +181,4 @@ def validate(self):
             return False
 
     def is_valid(self):
```

Comment:
Since now both `validate` and `is_valid` return the same value and do the same thing, we can remove one of them (`is_valid` seems like the best candidate to be removed).

This is a relatively new feature with not many users yet (accept me 😅), so we can delete it without considering backward compatability.